src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -673,7 +673,57 @@ describe('VAOS <VAFacilityPageV2> multiple facilities', () => {
     expect(screen.queryByText(/Facility 128/i)).not.to.exist;
   });
 
-  it('should show not supported message when direct is supported and not eligible, and requests are not supported', async () => {
+  it('should show past visits message when direct is supported and not eligible, and requests are not supported', async () => {
```

Comment:
Our facility page tests seem kinda disorganized at this point, I think I'm going to see if I can reorganize them into something that makes more sense in a follow up PR.